kratos/sources/kratos_application.cpp

```diff
@@ -202,6 +203,7 @@ void KratosApplication::RegisterKratosCore() {
     KRATOS_REGISTER_ELEMENT("Element3D2N", mElement3D2N)
     KRATOS_REGISTER_ELEMENT("Element3D3N", mElement3D3N)
     KRATOS_REGISTER_ELEMENT("Element3D4N", mElement3D4N)
+    KRATOS_REGISTER_ELEMENT("QuadElement3D4N", mQuadElement3D4N)
```

Comment:
This naming is inconsistent with all the other Kratos elements. This necessarily needs to be reverted.

Reply:
@rubenzorrilla has it been merged to Kratos core? I though I ma merging to local branch OpenFoam, not to master. If it is not true, than sorry I can revert it right now.

Reply:
You're right. I didn't note that. Nevertheless, I'd suggest to avoid the creation of this new element and use the already existing one.

Reply:
we will do that, thnx.